api/jsonrpc/server.go

```diff
@@ -139,3 +139,14 @@ func (j *JSONRPCServer) UnitPrices(
 	reply.UnitPrices = unitPrices
 	return nil
 }
+
+type GetABIArgs struct{}
+
+type GetABIReply struct {
+	ABIJSON string `json:"abi"`
```

Comment:
It has to be a string because we're comparing hashes. If it were marshaled differently, the hash would change, and the frontend wallet would reject the transaction when we implement ABI hash checking.